tests/test_console.py

```diff
@@ -0,0 +1,44 @@
+from bbc_meet_spotify.console import console
+from bbc_meet_spotify.music import Music
+from bbc_meet_spotify.playlist_parsing import PlaylistChoices
+from unittest.mock import patch, MagicMock, ANY
+
+
+def test_invalid_playlist_type():
+    try:
+        console(PlaylistChoices("invalid"))
+    except ValueError:
+        pass
+
+
+@patch("bbc_meet_spotify.console.BBCSounds")
```

Comment:
I'm not a Python expert so feel free to propose an alternative here if you'd prefer :)

Reply:
this seems like a good shout to me, thanks!